src/plots/gl3d/scene.js

```diff
@@ -380,6 +380,11 @@ function computeTraceBounds(scene, trace, bounds) {
                 }
             }
         }
+
+        if(trace.type === 'surface') {
+            bounds[0][d] += trace._objectOffset[d];
+            bounds[1][d] += trace._objectOffset[d];
+        }
```

Comment:
@archmoj as discussed just now let's get a clear test case (mock I guess) where `_objectOffset` matters - offhand this looks weird to me as we're recalculating min and max from the input data `trace.(x|y|z)` - which is redundant anyway since you already did this in `surface/convert` - and then adding the `_objectOffset` to it, which seems like double-counting the middle.